src/index.spec.js

```diff
@@ -336,6 +336,7 @@ describe('Dropzone', () => {
       );
       dropzone.find(ChildComponent).simulate('dragEnter', { dataTransfer: { files } });
       dropzone.simulate('dragEnter', { dataTransfer: { files } });
+      dropzone.simulate('dragEnter', { dataTransfer: { files } }); // make sure we de-dupe events
```

Comment:
Makes sense! I think it's fine not to allocate too much memory and save some work later. Let's stick to it.